code/citadel/cit_reagents.dm

```diff
@@ -153,14 +153,13 @@
 	if(prob(33))
 		if(M.getArousalLoss() >= 100 && ishuman(M) && M.has_dna())
 			var/mob/living/carbon/human/H = M
-			to_chat(H, "<span class='love'>Your libido is going haywire!</span>")
-			H.mob_climax(forced_climax=TRUE)
+			if(prob(50)) //Less spam
```

Comment:
It doesn't need to tell you that often, really. >20u of the stuff lasts a while anyway, so you'll still get the message a fair few times while it metabolizes.